modules/op-release-notes-1-14.adoc

```diff
@@ -0,0 +1,269 @@
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: TaskRun
+metadata:
+  name: remote-task-reference
+spec:
+  taskRef:
+    resolver: http
+    params:
+    - name: url
+      value: https://raw.githubusercontent.com/tektoncd-catalog/git-clone/main/task/git-clone/git-clone.yaml
+----
++
+.Example usage for a pipeline
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: http-demo
+spec:
+  pipelineRef:
+    resolver: http
+    params:
+    - name: url
+      value: https://raw.githubusercontent.com/tektoncd/catalog/main/pipeline/build-push-gke-deploy/0.1/build-push-gke-deploy.yaml
+----
+
+* With this update, you can use an enum declaration to limit the values that you can supply for a parameter of a pipeline or task. To use this feature, you must set the `pipeline.options.configMaps.feature-flags.data.enable-param-enum` spec to `true`.
++
+.Example usage
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: pipeline-param-enum
+spec:
+  params:
+  - name: message
+    enum: ["v1", "v2"]
+    default: "v1"
+# ...
+----
+
+* With this update, when using the Git resolver with the authenticated SCM API, you can override the default token, SCM type, and server URL that you configured.
++
+.Example usage
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: git-api-demo-tr
+spec:
+  taskRef:
+    resolver: git
+    params:
+    - name: org
+      value: tektoncd
+    - name: repo
+      value: catalog
+    - name: revision
+      value: main
+    - name: pathInRepo
+      value: task/git-clone/0.6/git-clone.yaml
+    # create the my-secret-token secret in the namespace where the
+    # pipelinerun is created. The secret must  contain a GitHub personal access
+    # token in the token key of the secret.
+    - name: token
+      value: my-secret-token
+    - name: tokenKey
+      value: token
+    - name: scmType
+      value: github
+    - name: serverURL
+      value: https://ghe.mycompany.com
+----
+
+* With this update, you can define default resource requirements for containers and init-containers in pods that {pipelines-shortname} creates when executing task runs. Use the `pipeline.options.configMaps.config-defaults.default-container-resource-requirements` spec in the `TektonConfig` CR to set these requirements. You can set the default values for all containers and also for particular containers by name or by prefix, such as `sidecar-*`.
+
+[id="operator-new-features-1-14_{context}"]
+=== Operator
+
+* With this update, {pipelines-shortname} supports horizontal pod autoscaling for the operator proxy webhook. If the pod that runs the operator proxy webhook reaches 85% CPU utilization, the autoscaler creates another replica of the pod. If you want to use more than one replica for the operator proxy webhook at startup, you must configure this number in the `options.horizontalPodAutoscalers` spec of the `TektonConfig` CR.
```

Comment:
Operator should be capitalized, I think - see https://redhat-documentation.github.io/supplementary-style-guide/#operator  I will only mark once here.